bkpmgmt/bkpmgmt.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import argparse
+import logging
+import sys
+
+LOG = logging.getLogger(__name__)
+LOG.addHandler(logging.StreamHandler())
+LOG.setLevel(logging.WARN)
+
+
+def main():
+    """Main function.
+    Manage cli arguments and call the corresponding function per subcommand
+    with the needed arguments.
+    """
+    parser = argparse.ArgumentParser(
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+        description="""\
+        Manage dirvish backups with an underlying zfs storage pool.
+        """,
+    )
+    parser.add_argument(
+        "command",
+        help="""\
+        All subcommands that modify zfs volumes and dirvish configurations.
+        See the manpage (man 8 bkpmgmt) for more inforation.
+        """,
+    )
+    parser.add_argument(
+        "-c", "--server",
```

Comment:
A bit confusing that the long argument is called `--server` and not `--client`. Maybe `-v` and `--vault` would be more descriptive?

Reply:
I think `vault` is cleaner since dirvish references the backup directories as vaults in their man page and the goal of bkpmgmt is to manage banks and vaults. But I see where you are going with `client`, this way you would automatically set the `client` setting in the configuration file.

Just nitpicking. Either way is fine for me.

Reply:
Ok, so we define it will be `-v` and `--vault`.